### ISSUE TITLE
VREffect.js: unwanted ghosting occurs when objects are added to the camera

### ISSUE THREAD

**Issue by skezo** [mrdoob/three.js#7041]
**Issue**
When adding an object (e.g. cube) to the camera while `VREffect.js` is active ghosting will occur on the object when the camera moves. This happens with master and dev versions (r.72) of `Three.js` and `VREffect.js` 

![ghosting](https://cloud.githubusercontent.com/assets/793287/9438741/46be1eea-4a93-11e5-9d89-0fd01cd50fa7.png)

To see the issue in action visit: http://misalabs.azurewebsites.net/ghosting/index.html using iPhone (iOS 8+ safari) or Android 4.4+ (Chrome). Press the "Cardboard" icon to activate VR mode. The live example is using [webvr-manager](https://github.com/borismus/webvr-boilerplate/) but the issue can be recreated using the below code.

``` javascript
var renderer = new THREE.WebGLRenderer({ antialias: true });
renderer.setPixelRatio(window.devicePixelRatio);
document.body.appendChild(renderer.domElement);

var scene = new THREE.Scene();
var camera = new THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.3, 10000);
//Add camera to scene
scene.add(camera)

// Apply VR headset positional data to camera.
var controls = new THREE.VRControls(camera);

// Apply VR rendering to renderer.
var effect = new THREE.VREffect(renderer);
effect.setSize(window.innerWidth, window.innerHeight);

// Create 3D object to attach to camera
var geometry = new THREE.BoxGeometry(0.5, 0.5, 0.5);
var material = new THREE.MeshNormalMaterial();
var cube = new THREE.Mesh(geometry, material);
cube.position.z = -3;

// Add cube to three.js camera
camera.add(cube);

//Animation Loop
function animate(timestamp) {
requestAnimationFrame(animate);
controls.update();
effect.render(scene, camera);
}
animate();

```

**Expected outcome**
The object added to the camera should appear fixed to the camera with no ghosting effect present while moving the camera.

![no-ghosting](https://cloud.githubusercontent.com/assets/793287/9438751/4f1f72a0-4a93-11e5-92e9-7abbeeebb182.png)

**Workaround:**
If I add `camera.updateMatrixWorld()` before the `effect.render(scene, camera)`  in the callback of the `requestAnimationFrame()` method the ghosting stops. For example:  

``` javascript
//Animation Loop
function animate(timestamp) {
requestAnimationFrame(animate);
controls.update();
camera.updateMatrixWorld();
effect.render(scene, camera); 
}
animate();
```

To see the workaround in action visit: http://misalabs.azurewebsites.net/ghosting/no-ghosting.html
 using iPhone (iOS 8 safari) or Android 4.4+ (Chrome). Press the "Cardboard" icon to activate VR mode. The live example is using [webvr-manager](https://github.com/borismus/webvr-boilerplate/) but the workaround can be recreated using the above code.

**Comment by WestLangley** [comment]
Yes. `VREffect.render()` is accessing `camera.matrixWorld()` without first updating it.

The proper fix is to add a call to `camera.updateMatrixWorld()` within `VREffect.render()`, instead.

However, this issue should be automatically resolved when #7019 is merged, and changes to `VREffect.render()` should not be required.

**Comment by brianchirls** [comment]
I see #7019 has been merged. Should this one be closed? Can someone confirm it's fixed?
